.github/CODEOWNERS

```diff
@@ -13,6 +13,4 @@
 #
 # For anything not explicitly taken by someone else:
 
-* @open-telemetry/go-maintainers @open-telemetry/collector-maintainers
-
-chloggen/ @open-telemetry/go-maintainers @open-telemetry/collector-maintainers @open-telemetry/collector-contrib-approvers
+* @open-telemetry/go-maintainers @open-telemetry/go-approvers @open-telemetry/collector-maintainers @open-telemetry/collector-approvers @open-telemetry/collector-contrib-maintainers @open-telemetry/collector-contrib-approvers
```

Comment:
```suggestion
* @open-telemetry/go-maintainers @open-telemetry/go-approvers @open-telemetry/collector-maintainers @open-telemetry/collector-approvers @open-telemetry/collector-contrib-maintainer @open-telemetry/collector-contrib-approvers
```

Reply:
Thanks for this. Interesting mistake :) we should look into fixing that and add an "s" maybe?